arrow/obsidianmodule.c

```diff
@@ -39,7 +39,10 @@ typedef struct {
   int reactions_count;
   int substrates_count;
   long *stoichiometry;
-  double *rates;
+  double *rates_flat;
```

Comment:
Love how many places you have to add this.... Looking forward to evaluating @1fish2's proposal of using cython for the glue (module) code here.